posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md

```diff
@@ -0,0 +1,457 @@
+6. Add basic auth by selecting `spring-boot-jwt-basic-auth` then press the `Apply New (2)` button.
+7. Add React with JWT support by selecting `react-jwt` and then press the `Apply New (3)` button.
+8. For convenience, add the front-end maven plugin by selecting `frontend-maven-plugin` and press the `Apply New (1)` button.
+9. You should now have 17 modules added and can press the `Download` button to get the generated application.
+10. Unzip the application in an empty directory and you are ready to customize!
+
+##  Run the application and check out the dummy-service
+
+You can run the application from your new directory by running `./mvnw` if you are on a mac/linux machine. You may need to give the file executable permissions. If you already have maven installed, you can run `mvn`.
+
+The base application will be available at http://localhost:8080 (using the port you chose during generation)
+
+If you go to http://localhost:8080/swagger-ui/index.html you can see that the dummy-feature service is called `Beers`. A GET, POST and DELETE action was generated. A beer is shown to include an `id`, `name` and `unitPrice`.
+
+## Add a Beers function in React
+
+In order to call the Beers service, we first need an interface that matches the attributes of a beer in the service.
+
+Create a folder under `src/main/webapp/app` called `beers`.
+
+Create a file in the beers folder called `interface.d.ts`
+
+```javascript
+export type BeersFunctionType = {
+   setBeers: Dispatch<Array<IBeer>>;
+};
+ export interface IBeer {
+   id: string;
+   name: string;
+   unitPrice: string;
+};
+```
+
+Once you have an interface, you can create a file called `beers.ts` in a folder under beers called `services`
+
+```javascript
+import axios from 'axios';
+ 
+import { getLocalStorage } from '@/common/services/storage';
+import { BeersFunctionType } from '@/beers/interface';
+ 
+const BEERS_URL = '/api/beers';
+ 
+export const getBeers = ({ setBeers }: BeersFunctionType) => {
+ const headers = {
+   'Authorization': 'Bearer ' + getLocalStorage('token')
+ }
+  return axios
+   .get(BEERS_URL, {
+     headers: headers
+   })
+   .then(response => {
+     setBeers(response.data.beers);
+   })
+   .catch(error => {
+     console.error(error);
+   });
+};
+```
+
+In the authentication code in the generated project, the JWT is stored in local storage during login. We then place this token in the header, in order to make the service call. The response from the service is returned in an array of beer objects.
+
+## Display the results of a Beers service call
+
+Create a `beers.tsx` file to output the results
+
+```javascript
+import { useState, useEffect } from 'react';
+import { getBeers } from '@/beers/services/beers';
+import { IBeer } from '@/beers/interface';
+ 
+export default function Beers() {
+   const [beers, setBeers] = useState<Array<IBeer>>([]);
+ 
+   useEffect(() => {getBeers({setBeers})}, []);
+ 
+   return (
+       <main style={{ padding: "1rem 0" }}>
```

Comment:
I would remove the inline styling. CSS styles don't really add anything to your code blocks other than clutter. Unless you wanted someone to literally be able to copy paste out of the blog and get it working.